08.ls_object/printer.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class Printer
+  require_relative 'file_detail'
+
+  NUMBER_OF_COLUMN = 3
+
+  def initialize(directory, long_format, files)
+    @long_format = long_format
+    @files_detail = files_detail_array(directory, files)
+  end
+
+  def files_detail_array(directory, files)
```

Comment:
```suggestion
  def files_details(directory, files)
```

複数形にして配列であることを表現すればOKです